boards/microchip-pic32mx-generic/misc/bootloader.hex

```diff
@@ -12,7 +12,7 @@
 :1000A0000100292500000A2444492A7D01608A4074

```

Comment:
Is this file checked in intentionally, or is it generated?

Reply:
It is checked in intentionally even though it is a compiler output. It somewhat difficult to compile (lots of tools needed) but sometimes you just want to get an emulator and flash it into the device.